cafebabel/templates/articles/tags/detail.html

```diff
@@ -31,7 +31,13 @@
 
 {% block body_class %}tag-page{% endblock %}
 {% block body %}
-  <div class=tag-header data-image={{ tag.image_url }}></div>
+  {% if tag.has_image %}
+    <div class="tag-header image">
```

Comment:
What is the goal of `image` vs. `default`? Don't you want to apply the parallax to the default too?

Reply:
when `.image`: `.tag-page .tag-header.image + header{ margin-top:60vh}`
when `.default`: `.tag-page .tag-header.image + header{ margin-top:30vh}`